modules/activities.js

```diff
@@ -0,0 +1,38 @@
+const activities = `

```

Comment:
Make this a function that returns a string literal containing your `HTML` code.

Reply:
Your module names should start with capital letters. `const Activities` instead of `const activities`